ts/dx.all.d.ts

```diff
@@ -1852,6 +1850,10 @@ declare module DevExpress.data {
          * [descr:ODataStore.createQuery(loadOptions)]
          */
         createQuery(loadOptions: any): any;
+        /**
+         * [descr:ODataStore.insert(values)]
+         */
+        insert(values: any): Promise<any> & JQueryPromise<any>;
```

Comment:
Known issue:
```suggestion
        insert(values: any): PromiseExt<any> & JQueryPromise<any>;
```
This will be fixed with another PR